bundle/rhpam/metadata/annotations.yaml

```diff
@@ -6,7 +6,7 @@ annotations:
   operators.operatorframework.io.bundle.package.v1: rhpam-kogito-operator
   operators.operatorframework.io.bundle.channels.v1: 8.x
   operators.operatorframework.io.bundle.channel.default.v1: 8.x
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.9.0+git
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.10.0+git
```

Comment:
are you sure about that change ?

Reply:
I think so, it is added by operator-sdk, in this case, operator-sdk version:
```
operator-sdk version: "v1.11.0", commit: "28dcd12a776d8a8ff597e1d8527b08792e7312fd", kubernetes version: "1.20.2", go version: "go1.16.7", GOOS: "linux", GOARCH: "amd64"
```